test/js/inline-style-property-map.js

```diff
@@ -0,0 +1,16 @@
+suite('Inline StylePropertyMap', function() {
+  setup(function() {
+    this.element = document.createElement('div');
+    document.body.appendChild(this.element);
+    this.element.style.opacity = '0.5';
+  });
+  teardown(function() {
+    document.body.removeChild(this.element);
+  });
+
+  test('Test that a new InlineStylePropertyMap is created when a HTML element referance calls the ' +
```

Comment:
This test seems suspicious - there is no InlineStylePropertyMap anymore? Perhaps it should be "Test that element.styleMap returns an Inline StylePropertyMap"

Also the formatting in this function is still weird...